tests/test_interfaces.py

```diff
@@ -14,19 +14,11 @@
 
 import unittest
 from random import randint
-import jsonschema
-
-from factoring.interfaces import factor
-from factoring.json_schema import json_schema
 
+from demo import factor
```

Comment:
I'm confused about the purpose of `test_factor_validity` below? If it's always skipped why do we need it? Or if we need it, then let's not always skip it.

Reply:
I unskipped it. There is a small probability of failure, so I give it three chances for each factor.